service/controller/clusterapi/v31/cloudconfig/tccp.go

```diff
@@ -54,17 +54,27 @@ func (t *TCCP) Render(ctx context.Context, cr cmav1alpha1.Cluster, clusterCerts
 		params = k8scloudconfig.DefaultParams()
 
 		params.Cluster = cmaClusterToG8sConfig(t.config, cr, labels).Cluster
+		// with network load-balancer, master needs to access API via localhost
+		// this is due to problem that network lb does not change source IP
+		// and packet sent from master via network load-balancer to master
+		// has same source and destination IP and the operating system will not forward the packet back to LB
+		// result is that connection does not work
+		params.Cluster.Kubernetes.API.Domain = Localhost
+		params.Cluster.Etcd.Domain = Localhost
 		params.DisableEncryptionAtREST = true
 		// Ingress controller service remains in k8scloudconfig and will be
 		// removed in a later migration.
+		params.DisableCalico = true
 		params.DisableIngressControllerService = false
 		params.EtcdPort = key.EtcdPort
 		params.Extension = &MasterExtension{
 			awsConfigSpec: cmaClusterToG8sConfig(t.config, cr, labels),
 			baseExtension: baseExtension{
-				cluster:       cr,
-				encrypter:     t.config.Encrypter,
-				encryptionKey: cc.Status.TenantCluster.Encryption.Key,
+				calicoMTU:      params.Cluster.Calico.MTU,
```

Comment:
Not sure what would need to be discussed, We had configurable MTU since a long time I am jsut passing the value to the right place here.

I know that we want to have it configurable per cluster in the future as some customer asks for it.

Reply:
No, nothing _to_ be discussed, but I was referring to that discussion on specific customer needs, but yeah. All good here. :+1: